prog/dftb+/lib_math/degeneracy.F90

```diff
@@ -222,9 +223,7 @@ subroutine degeneracyRanges(levelRange, nGrp, Levels, tol, grpMembership, levelS
       iStart = 1
       iEnd = nOrb
     end if
-    if (iStart >= iEnd) then
-      call error("Degeneracy range is broken, should not be here")
-    end if
+  @:ASSERT(iStart <= iEnd)
```

Comment:
The negation of `iStart >= iEnd` would be `iStart < iEnd`. What is the reason for using less equal here?

Reply:
A range of x:x is OK for a single non-degenerate level, so the original version of the test is broken.